README.md

```diff
@@ -95,6 +95,8 @@ int main() {
 
 For more examples cf. the [examples](examples) directory.
 
+*Note: Until the release APIs are expected to be undergoing (potentially breaking) changes. Binary data layout is stable, though*
```

Comment:
. is missing at the end of the sentence.

Reply:
I would move that close to the top of the file to ensure people actually see it. Yeah, and the dot.